packages/block-editor/src/components/warning/index.js

```diff
@@ -9,13 +9,28 @@ import clsx from 'clsx';
 import { DropdownMenu, MenuGroup, MenuItem } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { moreVertical } from '@wordpress/icons';
+import { useEffect, useRef } from '@wordpress/element';
 
 function Warning( { className, actions, children, secondaryActions } ) {
+	const alertRef = useRef( null );
+
+	useEffect( () => {
+		if ( null !== alertRef.current ) {
+			alertRef.current.focus();
+		}
+	}, [] );
```

Comment:
Question: can this be simplified? I'm not sure there's the need to set the initial ref to `null` and then checking for it. To my understanding the optional chaining operator `?.` would make this much more concise. See an example here:

https://github.com/WordPress/gutenberg/blob/e4649fe6896123231de35da0875e93ac2920c23a/packages/block-library/src/video/tracks-editor.js#L182-L186